### PR TITLE
Add pasta recipe

### PR DESCRIPTION
**Pull request by cornelius**

### CODE REVIEW

File: recipes/pasta.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+* Pasta
+* Tomatoes
+* Basil
+
+## Instructions
+
+* Cook pasta
+* Prepare sauce
+* Serve with grated cheese
```
Comment (mxmehl) [inline recipes/pasta.md:14]:
```suggestion
* Serve with grated cheese
* Enjoy!
```

**Review by mxmehl — CHANGES_REQUESTED** [review]
The most important step is missing

File: recipes/pasta.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Pasta with tomato sauce
+
+## Ingredients
+
+* Pasta
+* Tomatoes
+* Basil
```
Comment (cornelius) [inline recipes/pasta.md:7]: Please add garlic as well

**Review by mxmehl — APPROVED** [review]
Thanks for your amendments, it's great!

**Pull request merged**